### PR TITLE
Editorial: Various editorial updates

### PR DESCRIPTION
**Pull request by anba**
Various editorial updates, including changes to align with ECMA-262 conventions and some editorial bug fixes. 

---

The PR applies on top of https://github.com/tc39/ecma402/pull/826.

### CODE REVIEW

File: spec/negotiation.html [hunk 1]
```diff
@@ -442,14 +491,14 @@ <h1>PartitionPattern ( _pattern_ )</h1>
           1. Assert: _endIndex_ is greater than _beginIndex_.
           1. If _beginIndex_ is greater than _nextIndex_, then
             1. Let _literal_ be a substring of _pattern_ from position _nextIndex_, inclusive, to position _beginIndex_, exclusive.
```
Comment (ben-allen) [inline spec/negotiation.html:493]: appears to be an unnecessary inclusive/exclusive that got through

**Review by ben-allen — COMMENTED** [review]
whew! there's places where it seems like you missed changing all of the instances of the thing you were changing (i.e. an "append" that didn't get changed to "string-concatenation") but this is as far as I can tell as stellar as the rest of this.

I'm very tempted to put in a PR on 262 regularizing their spec as well, since there's oddities of the type you fixed still present there. or at least I will be tempted, next time I have copious free time.

**Review by ben-allen — APPROVED** [review]

**Comment by ben-allen** [comment]
Any objections to merging this one? rebasing is going to be more and more of a pain as time goes on. @sffc @bakkot @FrankYFTang @ryzokuken

**Comment by sffc** [comment]
Yes, if the editors approve, please merge; we shouldn't let PRs stay open this long

**Comment by ben-allen** [comment]
I've merged the rebased version of this PR and https://github.com/tc39/ecma402/pull/826 via the new https://github.com/tc39/ecma402/pull/893.

**Pull request closed without merging**